tests/resources/test_resources_communities.py

```diff
@@ -9,34 +9,73 @@
 
 import pytest
 
-# from invenio_rdm_records.proxies import current_rdm_records
+from invenio_rdm_records.proxies import current_rdm_records_service
 
 
-# @pytest.fixture()
-# def service():
-#     """Get the current RDM records service."""
-#     return current_rdm_records.records_service
+@pytest.fixture()
+def service():
+    """Get the current RDM records service."""
+    return current_rdm_records_service
 
 
-# @pytest.fixture()
-# def published(uploader, minimal_record, community, service, running_app, db):
-#     minimal_record["parent"] = {
-#         "review": {
-#             "type": "community-submission",
-#             "receiver": {"community": community.data["id"]},
-#         }
-#     }
+@pytest.fixture()
+def record(uploader, minimal_record, community, service, running_app, db):
+    """Record that belongs to a community."""
+    # Create draft with review
+    draft = service.create(uploader.identity, minimal_record)
+    # Publish
+    record = service.publish(uploader.identity, draft.id)
+    # TODO replace the following code by the service func that adds the record to a community
```

Comment:
Are these TODOs being tracked somehow? E.g. in other issues.

Reply:
@ntarocco is working on implementing the add to multiple communities, they should be fixed in the next PR after this one is merged, however it might be worth to create a card in the board to make sure that we check this